homeassistant/components/zwave_js/addon.py

```diff
@@ -100,12 +100,14 @@ class AddonManager:
     """Manage the add-on.
 
     Methods may raise AddonError.
-    Only one instance of this class may exist
+    Only one instance of this class may exist per add-on
     to keep track of running add-on tasks.
     """
 
-    def __init__(self, hass: HomeAssistant) -> None:
+    def __init__(self, hass: HomeAssistant, addon_name: str, addon_slug: str) -> None:
```

Comment:
Should we include a minimum version that we want the add-on to have or are we always going to push for latest possible?

Reply:
This is just a refactor. I haven't planned to add any new features here. That's something that can be discussed separately.

Reply:
A minimum version could be used to check if we have the required version. When we update though, I think we always want to go to the latest version. But I haven't thought about specific cases for other integrations yet. So I'll leave it at that for now.

Reply:
I guess the max version is already set by the addon having a min HA version. So HA can just go to latest.